webpack.config.js

```diff
@@ -46,7 +46,8 @@ function getPlugins() {
         plugins.push(new webpack.optimize.UglifyJsPlugin({
             compress: {
                 warnings: false
-            }
+            },
+            mangle: false
```

Comment:
Usage of `Function` force us to disable `mangle` on `UglifyJs`, it does not handle it.